Spikeinterface is causing Dev Tests to fail

Dev version of Spikeinterface is causing a couple tests to fail. Looks like typing shenanigans.

```
FAILED tests/test_ecephys/test_tools_spikeinterface.py::TestAddElectrodes::test_integer_channel_names - numpy.core._exceptions._UFuncNoLoopError: ufunc 'add' did not contain a loop with signature matching types (dtype('<U1'), dtype('int64')) -> None
FAILED tests/test_ecephys/test_tools_spikeinterface.py::TestAddUnitsTable::test_integer_unit_names_overwrite - numpy.core._exceptions._UFuncNoLoopError: ufunc 'add' did not contain a loop with signature matching types (dtype('<U1'), dtype('int64')) -> None
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -172,7 +172,7 @@ cellexplorer = [
 ]
 edf = [
     "neo>=0.13.3",
-    "pyedflib>=0.1.36,<0.1.39",  #
+    "pyedflib>=0.1.36,<0.1.39",  # Remove ceiling after neo 0.14.1 releases
     "spikeinterface>=0.101.2",
 ]
 intan = [

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -149,95 +149,95 @@ behavior = [
 ## Ecephys
 alphaomega = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 axona = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 biocam = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 blackrock = [
     "neo>=0.14",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 cellexplorer = [
     "hdf5storage>=0.1.18",
     "neo>=0.13.3",
     "pymatreader>=0.0.32",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
     "setuptools; python_version >= '3.12'"
 ]
 edf = [
     "neo>=0.13.3",
-    "pyedflib>=0.1.36",
-    "spikeinterface>=0.101.0",
+    "pyedflib>=0.1.36,<0.1.39",  #
+    "spikeinterface>=0.101.2",
 ]
 intan = [
     "neo>=0.14",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 kilosort = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 
 maxwell = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 mcsraw = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 mearec = [
     "MEArec>=1.8.0",
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
     "setuptools; python_version >= '3.12'"
 ]
 neuralynx = [
     "natsort>=7.1.1",
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 neuroscope = [
     "lxml>=4.6.5",
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 openephys = [
     "lxml>=4.9.4",
     "neo>=0.14",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 phy = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 plexon = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
     "zugbruecke >= 0.2.1; platform_system != 'Windows'",
 ]
 spike2 = [
     "neo>=0.13.3",
     "sonpy>=1.7.1; python_version=='3.9' and platform_system != 'Darwin'",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 spikegadgets = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 spikeglx = [
     "neo>=0.14",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 tdt = [
     "neo>=0.13.3",
-    "spikeinterface>=0.101.0",
+    "spikeinterface>=0.101.2",
 ]
 ecephys = [
     "neuroconv[alphaomega]",

